package.json

```diff
@@ -143,7 +143,7 @@
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "65.25 kB"
+      "maxSize": "65.80 kB"
```

Comment:
I just upgraded it to `65.5 kB` in my PR (https://github.com/algolia/instantsearch.js/pull/4592), should I remove it?

Reply:
@shortcuts you can keep it there because we don't know which PR gets merged first :)

Reply:
Right but if I override your limit it might fail your tests, unless your bundle size is < 65.5 kB

Reply:
If it fails, I will increase it again :)